### PR TITLE
Add Catalan translations

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added Catalan translations
```

## Subject

Added catalan translations :)

### CODE REVIEW

**Comment by jordisala1991** [comment]
Wait, do not merge there is a mistake

**Comment by jordisala1991** [comment]
Fixed now

File: src/Resources/translations/SonataMediaBundle.ca.xliff [hunk 1]
```diff
@@ -0,0 +1,568 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
```
Comment (greg0ire) [inline src/Resources/translations/SonataMediaBundle.ca.xliff]: `file.ext`?

Comment (greg0ire) [reply src/Resources/translations/SonataMediaBundle.ca.xliff]: what is this about

Comment (jordisala1991) [reply src/Resources/translations/SonataMediaBundle.ca.xliff]: I copied it from the `en` version.

I fixed the catalan one and also all the other files :)

**Comment by jordisala1991** [comment]
Please review @sonata-project/contributors

**Comment by core23** [comment]
Thanks @jordisala1991

**Pull request merged**